bigbluebutton-html5/imports/ui/stylesheets/variables/focus.scss

```diff
@@ -20,6 +29,8 @@ $list-item-bg-hover: darken($user-list-bg, 7%);
 %customChatItemFocus {
```

Comment:
Why this focus is not handled on the `chat-item` style itself? Same question for the other placeholders.

Reply:
Only makes sense to have them in a shared file and using placeholders if you want to use them in multiple styles/components.